params/config_test.go

```diff
@@ -0,0 +1,12 @@
+package params
```

Comment:
We may need to add a copyright for this file

Reply:
I added it. Thank you